WordPress/Classes/ViewRelated/Blog/Site Settings/DiscussionSettingsViewController.swift

```diff
@@ -17,21 +19,16 @@ open class DiscussionSettingsViewController: UITableViewController {
     // MARK: - View Lifecycle
     open override func viewDidLoad() {
         super.viewDidLoad()
+
         setupNavBar()
         setupTableView()
-        setupNotificationListeners()
     }
 
     open override func viewWillAppear(_ animated: Bool) {
         super.viewWillAppear(animated)
+
         tableView.reloadSelectedRow()
         tableView.deselectSelectedRowWithAnimation(true)
-        refreshSettings()
```

Comment:
My gut feeling is this refreshing settings call is here for a reason. Is there any straightforward way to keep this call?

Reply:
Yeah, it's safe to keep it. It doesn't really need to be part of this PR.
Reverting..